packages/teleport-postprocessor-js-beautify-html/src/index.ts

```diff
@@ -0,0 +1,32 @@
+// @ts-ignore
+import { html as beautifyHTML } from 'js-beautify'
+
+import { PostProcessor, PrettierFormatOptions, FileType } from '@teleporthq/teleport-types'
+
+interface PostProcessorFactoryOptions {
+  fileType?: string
+  formatOptions?: PrettierFormatOptions
+}
+
+export const createPostProcessor = (options: PostProcessorFactoryOptions = {}) => {
+  const fileType = options.fileType || FileType.HTML
+
+  const processor: PostProcessor = (codeChunks) => {
+    if (codeChunks[fileType]) {
+      const result = beautifyHTML(codeChunks[fileType], {
+        wrap_line_length: 80,
```

Comment:
can we extract this into constants? same as prettier settings

Reply:
yup 👍